css/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  line-height: 1.5;
+}
+
+header {
+  font-size: 20px;
```

Comment:
I think you perhaps do not need to add this unless maybe we want to make the header have a bigger font size?
This is because, following the DOM structure, by default, the header inherits the size of the body. 

Maybe if you wanted a bigger font-size for the header such as 2rem using fluid measurements which is the same as 32px?